pkgs/misc/apulse/pressureaudio.nix

```diff
@@ -0,0 +1,81 @@
+{ stdenv, apulse, libpulseaudio, pkgconfig, intltool, autoreconfHook }:
+
+stdenv.mkDerivation {
+  name = "libpressureaudio-${apulse.version}";
+
+  src = libpulseaudio.src;
+
+  nativeBuildInputs = [ pkgconfig intltool autoreconfHook ];
+
+  configurePhase = ''
+    echo "Doing nothing."
+  '';
+
+  buildPhase = ''
+    echo "Building zero lines of code."
+  '';
+
+  installPhase = ''
+    echo "Coping libraries."
```

Comment:
Either use `:` for empty phases and reduce verbosity here, or fix the typo: ‘‘Coping’’ → ‘‘Copying’’